bc-rest/src/main/java/org/jboss/da/bc/BuildConfigurationsProduct.java

```diff
@@ -61,4 +79,25 @@ public ProductFinishResponse finishAnalyse(@ApiParam(
     protected BuildConfigurationsFacade<ProductInfoEntity> getFacade() {
         return bcpf;
     }
+
+    private <T> Response validation(T object, String message) {
+        Validations validations = new Validations();
+        Set<ConstraintViolation<T>> validate = validator.validate(object);
+        if (!validate.isEmpty()) {
+            for (Iterator<ConstraintViolation<T>> iterator = validate.iterator(); iterator
+                    .hasNext();) {
+                ConstraintViolation<T> cv = iterator.next();
+                ValidationField validation = new ValidationField(cv.getPropertyPath().toString(),
+                        cv.getMessage());
+
+                validation.setValue(cv.getInvalidValue().toString());
+                validations.addValidationField(validation);
+            }
+            return Response
+                    .status(Response.Status.BAD_REQUEST)
+                    .entity(new ErrorMessage(ErrorMessage.eType.INPUT_VALIDATION, message,
+                            validations)).build();
+        }
+        return null; // no errors
+    }
```

Comment:
Can this be extracted to standalone bean? So it's not copied in all other classes.
And it could maybe return Optional<Response> instead of nullable Response.